### DISCUSSION TITLE
[Feature]: Support for TV (make controls focusable)

### DISCUSSION THREAD

**Discussion by moskalakamil** [TheWidlarzGroup/react-native-video#4992] [Ideas]
### What problem are you trying to solve?

We currently do not support TV controls focus management. iOS controls work out of the box, but Android controls do not.

### Proposed solution

Support for TV (make controls focusable).

### Platforms

Apple tvOS, Android TV

#### Support

If this functionality is important to you and you need it, contact [TheWidlarzGroup](https://thewidlarzgroup.com/) - [hi@thewidlarzgroup.com](mailto:hi@thewidlarzgroup.com)

[a comment by an automated account is not shown]

**Comment by TWG-AI** [comment]
Thanks for raising this! Android TV focus management for built-in controls is indeed missing — tvOS works because the native controls handle focus, but Android TV requires explicit D-pad / remote focus wiring. We've noted this for the team.
